operations/before.go

```diff
@@ -95,6 +97,23 @@ var (
 
 		return catcher.Resolve()
 	}
+
+	autoUpdateCLI = func(c *cli.Context) error {
+		confPath := c.String("conf")
+		conf, err := NewClientSettings(confPath)
+		if err != nil {
+			return errors.Wrap(err, "problem loading configuration")
+		}
+		if conf.AutoUpgradeCLI {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+			if err = checkAndUpdateVersion(conf, ctx, true, false, true); err != nil {
+				fmt.Println("Automatic CLI update failed, continuing command execution")
```

Comment:
Will this actually continue command execution if we return an error? (Have we tested that?)

Reply:
It would return early and not execute the command if the download failed for some reason. I get why this would be problematic, better to allow the user to continue on with the old version rather than manually removing auto_upgrade_cli from their config yaml. I've removed the return err statement and now we'll just notify in the logs that the download failed in the off case that it does fail.